### PR TITLE
nits: found during textile integration

### PR DESCRIPTION
**Pull request by sanderpick**
Just a couple small things I encountered when combining this lib with `textile`.

### CODE REVIEW

File: api/client/client.go [hunk 1]
```diff
@@ -31,9 +31,12 @@ type ListenEvent struct {
 }
 
 // NewClient starts the client
-func NewClient(host string, port int) (*Client, error) {
-	url := fmt.Sprintf("%v:%v", host, port)
-	conn, err := grpc.Dial(url, grpc.WithInsecure())
+func NewClient(maddr ma.Multiaddr) (*Client, error) {
```
Comment (sanderpick) [inline api/client/client.go]: In keeping with using multiaddress everywhere...

File: api/server.go [hunk 2]
```diff
@@ -18,7 +18,7 @@ import (
 )
 
 var (
-	log = logging.Logger("api")
+	log = logging.Logger("threadsapi")
```
Comment (sanderpick) [inline api/server.go]: More specific :)

File: eventstore/options.go [hunk 3]
```diff
@@ -11,7 +11,7 @@ import (
 )
 
 const (
-	defaultDatastorePath = "datastore"
+	defaultDatastorePath = "eventstore"
```
Comment (sanderpick) [inline eventstore/options.go]: Also, more specific :)

File: api/client/client.go (hunk 1 above)
Comment (carsonfarmer) [reply api/client/client.go]: Yeh really nice, I like this. Should just get this into our heads as the default always.

**Pull request merged**